todo_app/Mongo_items.py

```diff
@@ -0,0 +1,129 @@
+import pymongo
+import os, datetime
+from mongo_config import Config
+from Mongo_db import get_db
+from bson.objectid import ObjectId
+
+db = get_db(Config.MONGO_DB)
```

Comment:
This line here occurs when the file is first imported; which means it happens before your integration test is able to mock this call out. A simple solution would be moving this call inside your functions - they can each access the DB when required
(Under the hood, this should use the same connection pool, so isn't a big overhead we should preemptively worry about)

Reply:
Thanks for all these comments -  I can't believe I was so close to getting this working - frustrating!